sys-apps/groff/groff-1.22.4.ebuild

```diff
@@ -40,6 +41,7 @@ PATCHES=(
 
 src_prepare() {
 	default
+	eautoreconf
```

Comment:
Why the `eautoreconf` call?

Reply:
https://github.com/gentoo/gentoo/pull/10697#discussion_r244044381